src/components/Picker.js

```diff
@@ -311,7 +311,7 @@ function Picker({
         const children = items.filter(item => item[_schema.parent] !== undefined && item[_schema.parent] !== null);
```

Comment:
I wonder if on this line here it should not already refer to sortedItems as well? That's a sincere question, not a strong feeling - you are in here working + really thinking + testing it, do you have an opinion?

```suggestion
        const children = sortedItems.filter(item => item[_schema.parent] !== undefined && item[_schema.parent] !== null);
```

Reply:
Ah, thought you were right first, but at the time children is initialized sortedItems contains only those items where parents is undefined or null (basically only the parents), so this would be always an empty array then. I think its correct now as is.
Cool to know! Sounds like you have some insider knowledge there :) Really been looking forward to the full turbomodules integration and I'm a big fan of flipper.

Reply:
re: react-native not insider knowledge, I just maintain enough (and large enough) modules in react-native ecosystem that it pays to participate in the release process and solve problems before releases whereas for most it does not pay to spend time that early. It's in public discussions on react-native-community/releases and the react-native discord server though. Came up while troubleshooting a javascript debugger regression on 0.65 (and 0.66)

Anyway, if you've tested both styles and my comment is not correct, then we should definitely go as is - thanks for checking